src/components/common/ContextualSaveBar.jsx

```
@@ -0,0 +1,307 @@
+          </button>
+
+          {/* Enhanced Save button with animation */}
+          <button
+            onClick={onSave}
+            disabled={isSaving}
+            className={`group relative px-6 py-2.5 font-bold rounded-lg transition-all duration-300 hover:scale-105 hover:shadow-xl border-2 disabled:cursor-not-allowed ${
+              isSaving
+                ? "bg-green-500 text-white border-green-400 animate-pulse"
+                : "bg-white hover:bg-gray-100 text-gray-900 border-white/20"
+            }`}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)